src/client/AM.PA.MonitoringTool/SlackTracker/Data/SlackModel/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Created by Rohit Kaushik (f20150115@goa.bits-pilani.ac.in) at the University of Zurich
+// Created: 2018-07-09
+// 
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SlackTracker.Data.SlackModel
+{
+    class User
+    {
+        public string id { get; set; }
+        public string team_id { get; set; }
+        public string name { get; set; }
+        public string real_name { get; set; }
```

Comment:
it's also a small detail, but in C# we (usually) don't use underscores, but camel-case. Here, the property should be called RealName.